workspace/postCode.ts

```diff
@@ -0,0 +1,30 @@
+import  { Coordinates } from './coordinates'
+
+const request = require('request')
+
+export class PostCode {
+    getCoordinates(): Promise<Coordinates> {
+        let promisePostCode: Promise <string> = new Promise((resolve) => {
+            var requestString = "https://api.postcodes.io/postcodes/" + this.code 
+            request(requestString, function (error, response, body) {
+              resolve(body); 
+            });
+        });
+        return promisePostCode.then((bodyString: string) => {
+            var postcodeArray = JSON.parse(bodyString);
+            var coords = new Coordinates(postcodeArray.result.latitude.toString(), postcodeArray.result.longitude.toString());
+            return coords;
+        });
+    }
+    
+    constructor(public code: string){
+
+    }
```

Comment:
Okay, so while this isn't _wrong_, it's quite surprising to not find the constructor as the first method (if the method was any longer, my assumption would be that it didn't have one!).

I know I'm pulling out what appear to be little things fairly frequently, but programmers have to keep a _lot_ in their heads at once, and reducing mental energy working out little things like this actually saves a surprising amount of work, which is the reason for all the consistent styling stuff existing.